### PR TITLE
[UICIRC-203 & UICIRC-206] - Refine loan and request related notice policies

### PR DESCRIPTION
**Pull request by maximdidenkoepam**
# Purpose
Update common components responsible for editing and displaying loan and request related notices.

# Links
https://issues.folio.org/browse/UICIRC-203
https://issues.folio.org/browse/UICIRC-206

# Screenshots
screen shot 2019-03-06 at 17 17 12
screen shot 2019-03-06 at 17 17 26

### CODE REVIEW

File: src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js [hunk 1]
```diff
@@ -96,99 +87,108 @@ class NoticeCard extends React.Component {
...
+                            </KeyValue>
+                          </Col>
+                        </React.Fragment>
+                      )}
+                    </Row>
+                  </KeyValue>
+                </Col>
+              </Row>
+              <Row>
+                <Col xs={12} className={css.fieldLabel}>
```
Comment (VictorSoroka1) [inline src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js]: Should be props on multiple lines?

File: src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js [hunk 2]
```diff
@@ -96,99 +87,108 @@ class NoticeCard extends React.Component {
...
-                      </KeyValue>
-                    </Col>
-                  </Row>
-                </KeyValue>
-              </Col>
-            </Row>
+          { notice.sendOptions.isTimeBasedEventSelected(timeBasedEventsIds) && (
+            <React.Fragment>
+              <Row>
+                <Col xs={12} className={css.fieldLabel}>
```
Comment (VictorSoroka1) [inline src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js]: Should be props on multiple lines?

File: src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js [hunk 3]
```diff
@@ -96,99 +87,108 @@ class NoticeCard extends React.Component {
...
-            <Col xs={3} className={css.noticeField}>
+            <Col xs={2} className={css.noticeField}>
               <KeyValue
                 label={<FormattedMessage id={`${translationNamespace}.notices.format`} />}
-                value={this.getTranslatedDropdownValue('format', i, noticesFormats)}
+                value={this.getTranslatedDropdownValue('format', noticesFormats)}
               />
             </Col>
-            <Col xs={3} className={css.noticeField}>
+            <Col xs={5} className={css.noticeField}>
```
Comment (VictorSoroka1) [inline src/settings/NoticePolicy/components/DetailSections/components/NoticeCard/NoticeCard.js]: Should be props on multiple lines in that file?

File: src/constants.js [hunk 4]
```diff
@@ -155,19 +140,47 @@ export const noticesIntervalPeriods = [
...
+export const requestTimeBasedEventsIds = {
+  HOLD_EXPIRATION: 'Hold Expiration',
+};
+
+export const requestItemStateChangeEventsIds = {
+  AVAILABLE: 'Available',
+};
+
+export const requestNoticesTriggeringEvents = [
+  { value: requestUserInitiatedEventsIds.RECALL_LOANEE, label: 'ui-circulation.settings.noticePolicy.requestNotices.recallLoanee' },
```
Comment (OleksandrAntonenko1) [inline src/constants.js]: Please split this into multiple lines.

**Pull request merged**